app/Livewire/Admin/LicenseManager.php

```
@@ -0,0 +1,126 @@
+    public $supportEndDate;
+
+    public $editingLicense = false;
+
+    public $deleteId;
+
+    public $showDeleteModal = false;
+
+    // meta
+    public string $metaTitle = 'LicenseManager';
+
+    public function viewOrderItem($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)